com.helix_editor.Helix.yml

```diff
@@ -0,0 +1,86 @@
+app-id: com.helix_editor.Helix
+
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+command: hx
+
+modules:
+  - name: hx
```

Comment:
this being the main module. it should be last.

also any chance to build from source?

Reply:
Might as well build from source, no problem with that. I don't know what are the best practices here, if building from source is better then I'll do that, but I think it's easier to just use prebuilt things

Reply:
building from source is always preferred.

Reply:
Ok so I tried to make it build from source, but I don't think it's worth it. We need to build tree-sitter grammars. And for that we need to fetch sources for them. So we will have to fetch pkgs.tspm.io for sources and generate manifest module source definitions from them, and then compile them. It just better to use prebuilt grammars. But if we use prebuilt grammars, we might as well use the prebuilt binary